packages/base/card-api.gts

```diff
@@ -2337,15 +2337,15 @@ export async function getIfReady<T extends CardBase, K extends keyof T>(
 
 export function getFields(
   card: typeof CardBase,
-  opts?: { usedFieldsOnly?: boolean; includeComputeds?: boolean }
+  opts?: { usedLinkedFieldsOnly?: boolean; includeComputeds?: boolean }
```

Comment:
the name of this property doesn't really convey the correct meaning. I think it should be `useContainedFieldsOnly`. there is never a situation where we would only use linked fields.

Reply:
my bad--this is actually not too terrible of wording--but the `only` is throwing me off. maybe a better phrase would be `includeUsedLinkedFields` (because we want to include contained fields as well).

Reply:
renamed

Reply:
I'm not sure about that wording. If `includeUsedLinkedFields` is false, does it mean we don't return the used linked fields? I think what we want is to include only used linked fields if that option is true and include all linked fields (used and unused) if that option is false.

```
let pendingFields = new Set<string>(
    Object.keys(
      getFields(model, {
        includeComputeds: true,
        includeUsedLinkedFields: !opts?.recomputeAllFields,
      })
    )
  );
```

Reply:
I think `excludeUnusedLinkedFields` is better.